src/Ilios/AuthenticationBundle/Resources/config/voters.yml

```diff
@@ -56,8 +56,6 @@ services:
         class: Ilios\AuthenticationBundle\Voter\CourseVoter
```

Comment:
If I do that then the other service can't use it as a parent and gave to define their own arguments.